examples/python/simple_local/run.py

```diff
@@ -16,19 +16,23 @@
 """
 
 import sys
-from pathlib import Path
 
 import cv2
-from openvino.model_zoo.model_api.models import (
+from openvino.model_api.models import (
     Classification,
     DetectionModel,
     SegmentationModel,
 )
 
 from PIL import Image
 
-sys.path.insert(0, str(Path(__file__).resolve().parents[1] / "tools/model_tools/src"))
-sys.path.insert(0, str(Path(__file__).resolve().parents[1] / "demos/common/python"))
+
+def from_local_xml(image):
```

Comment:
Can you please resolve a conflict? Sorry about that.

Reply:
Done. I also replaced a link to omz_tools to match https://github.com/openvinotoolkit/open_model_zoo/pull/3700 which we are going to merge